pallets/gear/src/manager/journal.rs

```diff
@@ -190,6 +190,21 @@ where
 
         let _ = common::waiting_init_take_messages(id_exited);
 
+        let active_program = common::get_active_program(id_exited.into_origin())
+            .expect("`exit` can be called only from active program; qed");
```

Comment:
paste here unreachable please, as it done everywhere

Reply:
What's the diff?